.Libplanet.Extensions.ActionEvaluatorCommonComponents/ActionEvaluationMarshaller.cs

```diff
@@ -42,4 +43,27 @@ public static ICommittedActionEvaluation Deserialize(byte[] serialized)
         var decoded = Codec.Decode(serialized);
         return Unmarshal(decoded);
     }
+
+    private static Exception? GetException(Dictionary dictionary)
+    {
+        if (dictionary["exception"] is not Text typeName)
+        {
+            return null;
+        }
+
+        Type? type = Assembly.GetCallingAssembly().GetType(typeName.Value);
+        if (type is null)
+        {
+            throw new ArgumentException($"Type {typeName.Value} not found in the calling assembly.");
```

Comment:
This line may need to return `null` or an alternative `Exception` type instance because if there is no type corresponding to `typeName`, it cannot unmarshal all `ActionEvaluation`, since `ActionEvaluationMarshaller.Unmarshal` will fail.